fiftyone/core/metadata.py

```diff
@@ -75,6 +78,70 @@ def _build_for_url(cls, url, mime_type=None):
         return cls(size_bytes=size_bytes, mime_type=mime_type)
 
 
+class SceneMetadata(Metadata):
+    """Class for storing metadata about 3D scene samples.
+
+    Args:
+        size_bytes (None): the size of scene definition and all children
+            assets on disk, in bytes
+        mime_type (None): the MIME type of the image
```

Comment:
nit: add more clarification on this arg?

Reply:
ah yes copy pasta. It is ignored and always set to `application/octet-stream` (didn't know what else to do)